util/generate_spdx_license_list.rb

```diff
@@ -0,0 +1,21 @@
+require 'json'
+require 'net/http'
+
+json = Net::HTTP.get('spdx.org', 'licenses/licenses.json')
+licenses = JSON.parse(json)['licenses'].map do |licenseObject|
+  licenseObject['licenseId']
+end
```

Comment:
I amended to reflect @goneall's [changes in the format of the JSON file designed to better mimic the SPDX RDF spec](https://github.com/spdx/tools/pull/2#issuecomment-102659566) in spdx/tools#2.